src/lightning_app/components/multi_node/pytorch_spawn.py

```diff
@@ -88,7 +88,7 @@ def run(
         elif world_size > 1:
             raise Exception("Torch distributed should be available.")
 
-        work_run(world_size, node_rank, global_rank, local_rank)
+        return work_run(world_size, node_rank, global_rank, local_rank)
```

Comment:
This won't work. We would need to use queues to bring back the work state and the return value. 
Do you want to try to address it ?

Reply:
that's only for standalone testing here. I am aware that with mp.spawn this won't work that way.